src/Tvheadend.h

```diff
@@ -488,15 +488,18 @@ class CTvheadend
   }
   inline const char *GetServerName    ( void )
   {
-    return m_conn.GetServerName();
+    static std::string serverName = m_conn.GetServerName();
```

Comment:
I was supposed to move the conversion from `std::string` to char pointers all the way up to `client.cpp` but I guess I forgot. It has to be static at some point, otherwise we'll return a dangling stack pointer if we return `blabla.c_str()`.